src/Compilers/CSharp/Portable/Symbols/Source/SourceMethodSymbolWithAttributes.cs

```diff
@@ -433,8 +439,17 @@ internal sealed override ObsoleteAttributeData ObsoleteAttributeData
 #if DEBUG // Can remove ifdefs and replace with Conditional after https://github.com/dotnet/roslyn/issues/47463 is fixed
                 static void verifyDataConsistent(CommonMethodEarlyWellKnownAttributeData? earlyData, MethodWellKnownAttributeData? lateData)
```

Comment:
>verifyDataConsistent [](start = 28, length = 20)

Consider inlining.

Reply:
I'd prefer to leave it as is, I feel that it's more readable, and it would still need `#if DEBUG`s for the `if` blocks.

---
In reply to: [488853166](https://github.com/dotnet/roslyn/pull/47651#discussion_r488853166) [](ancestors = 488853166)